### DISCUSSION TITLE
duda de TP4 - ingreso de valores

### DISCUSSION THREAD

**Discussion by rinrot** [INGCOM-UNRN-P1/cursada-2023#122] [Preguntas y respuestas] [answered]
hola! tengo una duda sobre las consignas del tp4¿es necesario que le dejemos al usuario ingresar los valores de las variables? se que en la primera se especifica que no es necesario, pero en los otros ejercicios no dice nada sobre el ingreso de los valores.

**Comment by martinvilu** [comment, marked as answer]
Si no permitís el ingreso de valores, que los que usas, sea varios como para probar la mayor cantidad de casos.
